tests/test_AFC_lane.py

```
@@ -3996,4 +3996,57 @@ def test_exception_in_prep_load_does_not_block_other_lanes_afterward(self):
+        lane.drive_stepper = MagicMock()
+        lane.move_advanced = MagicMock()
+        lane.homing_overshoot = 10
+        lane.homing_delta = 300
+        lane.unit_obj = MagicMock()
+        lane.get_speed_accel = MagicMock(return_value=(50, 100))
+        lane.get_active_assist = MagicMock(return_value=None)
+        return lane
+
+    def test_tool_endstop_falls_back_to_plain_move_for_virtual(self):
+        lane = self._make_lane_with_drive_stepper("virtual")
+        homed, dist, warn = lane.move_to(100, SpeedMode.SHORT,
```

Ruff (0.16.2)

[warning] Unpacked variable `dist` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)